doc/source/reference/c-api/types-and-structures.rst

```diff
@@ -1130,6 +1134,10 @@ PyUFunc_Type and PyUFuncObject
        A function which resolves the types and fills an array with the dtypes
        for the inputs and outputs
 
+       .. c:type:: PyUFunc_TypeResolutionFunc
```

Comment:
Indentation?

Reply:
PyUFunc_TypeResolutionFunc is not itself a member of the struct, so added one extra indentation.

Reply:
Again, the [indentation is rendered correctly](https://output.circle-artifacts.com/output/job/971c751b-d94a-4943-a791-32c2f46ad6d7/artifacts/0/doc/build/html/reference/c-api/types-and-structures.html#c.PyUFuncObject.type_resolver.PyUFunc_TypeResolutionFunc), but I wonder if this is the correct place for the type declaration. The only other mention I could find for `PyUFunc_TypeResolutionFunc` is in [NEP 40](https://numpy.org/neps/nep-0040-legacy-datatype-impl.html#dtype-handling-in-universal-functions), where it is mentioned but not defined. It will default to [`PyUFunc_DefaultTypeResolver`](https://github.com/numpy/numpy/blob/441c9e600be16e1619ef6fb04e7ecdf6a12069cc/numpy/_core/src/umath/ufunc_type_resolution.c#L274), which is part of the NumPy UFUNC_API but not documented.